### ISSUE TITLE
Box Render : Fatal error: Cannot use result of built-in function in write context in ...src\endobox\Box.php on line 140

### ISSUE THREAD

**Issue by ghost** [younishd/endobox#24]
Box Render method fatal error on php 7.2.6:

```
    /**
     * Render the box and everything attached to it then return the result.
     */
    public function render() : string
    {
        // assign data if any
        if (\func_num_args() > 0) {
            $this->assign(\func_get_args()[0]);
        }
    ...
```

However, this works if I remove the \ :

```
$this->assign(func_get_args()[0]);
```

or if I do:
```
$var = \func_get_args()[0];
$this->assign($var);
```

Meh

**Comment by younishd** [comment]
You're using an old version of endobox. This was fixed in [2017](https://github.com/younishd/endobox/commit/64bc6e6723fc6df011340198c63f46388bb84781) already.

Edit: The fix is not in the latest release. Next release will close this issue.

**Comment by ghost** [comment]
Ok thx

**Comment by younishd** [comment]
Fixed in a4b9e6aaf954c53c29d4c4e2c1d7a4f8528d0d69
